api/health/health.go

```diff
@@ -16,6 +16,8 @@ import (
 
 var _ Health = (*health)(nil)
 
+const GlobalTag = "global"
```

Comment:
nit. Also can we comment what it means for a check to have the global tag? 
```suggestion
const GlobalTag = "global"

var _ Health = (*health)(nil)
```

Reply:
changed